app/views/groups/_content.html.erb

```diff
@@ -48,13 +48,24 @@
               <th>Name</th>
               <th>Auth Employee Name</th>
               <th>Auth Employee Id</th>
+              <th>SubGroups</th>
             </tr>
         </thead>
         <% @groups.each do |group| %>
           <tr data-id=<%=group.id%> onclick="window.location='/groups/' + <%=group.id%>" >
             <td><%= group.name %></td>
             <td><%= group.user.name %></td>
             <td><%= group.user.employee_id %></td>
+            <td> 
+            <% count = 1 %>
+            <% @subgroups.each do |subgroup| %>
+            <% if group.id == subgroup.group.id %>
```

Comment:
properly indent the code... do end is block... if end is another block , it can be like
@subgroups.each do |subgroup|
    if group.id == subgroup.group.id
        ....
    end
end